src/components/Dropdown/Dropdown.story.jsx

```diff
@@ -1 +1,113 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import {
+  ChartColumnFloating32,
+  ChartLineData32,
+  ChartBubble32,
+  ChartVennDiagram32,
+  TableSplit32,
+  QqPlot32,
+  ChartEvaluation32,
+  ChartSunburst32,
+  Diagram32,
+} from '@carbon/icons-react';
+
+import DropdownWithIcon from './DropdownWithIcon';
+
+export const items = [
+  {
+    id: 'option-0',
+    icon: ChartColumnFloating32,
+    text: 'Column Chart',
+  },
+  {
+    id: 'option-1',
+    icon: ChartBubble32,
+    text: 'Bubble Chart',
+  },
+  {
+    id: 'option-2',
+    icon: ChartVennDiagram32,
+    text: 'Venn Diagram',
+  },
+  {
+    id: 'option-3',
+    icon: ChartEvaluation32,
+    text: 'Evaluation Chart',
+  },
+  {
+    id: 'option-4',
+    icon: ChartLineData32,
+    text: 'Line Chart',
+  },
+  {
+    id: 'option-5',
+    icon: TableSplit32,
+    text: 'Option 5',
+  },
+  {
+    id: 'option-6',
+    icon: QqPlot32,
+    text: 'QQ Plot',
+  },
+  {
+    id: 'option-7',
+    icon: ChartSunburst32,
+    text: 'Sunburst Chart',
+  },
+  {
+    id: 'option-8',
+    icon: Diagram32,
+    text: 'Diagram',
+  },
+];
+
 export { default as DropdownStory } from 'carbon-components-react/lib/components/Dropdown/Dropdown-story';
+
+const directions = {
+  'Bottom (default)': 'bottom',
+  'Top ': 'top',
+};
+
+const props = () => ({
+  id: text('Dropdown ID (id)', 'carbon-dropdown-example'),
+  direction: select('Dropdown direction (direction)', directions, 'bottom'),
+  label: text('Label (label)', 'Dropdown menu options'),
+  ariaLabel: text('Aria Label (ariaLabel)', 'Dropdown'),
+  disabled: boolean('Disabled (disabled)', false),
+  light: boolean('Light variant (light)', false),
+  titleText: text('Title (titleText)', 'Dropdown label'),
+  helperText: text('Helper text (helperText)', 'This is some helper text.'),
+  invalid: boolean('Show form validation UI (invalid)', false),
+  invalidText: text(
+    'Form validation UI content (invalidText)',
+    'A valid value is required'
+  ),
+});
+
+storiesOf('Watson IoT/Dropdown', module).add('with icons and labels', () => {
+  return React.createElement(() => {
+    const [selectedViewId, setSelectedViewId] = useState(items[1].id);
```

Comment:
selectedOption instead of selectedView?

Reply:
renamed to selectedItem to keep in line with all the other item names